docs/resiliency/index.asciidoc

```diff
@@ -152,6 +152,19 @@ We have ported the known scenarios in the Jepsen blogs that check loss of acknow
 The new tests are run continuously in our testing farm and are passing. We are also working on running Jepsen independently to verify
 that no failures are found.
 
+[float]
+=== Replicas can fall out of sync when a primary shard fails (STATUS: ONGOING)
+
+In a scenario with more than one replica and a primary shard fails, one of the
+replicas will be promoted to be the primary shard. In this scenario, it is
+possible for the remaining replicas to be out of sync with the new primary
+shard. This arises due to the fact that indexing in Elasticsearch occurs
```

Comment:
I think it's not fully evident why concurrency plays a role here. Even without concurrency the replica can mis (a single) op from the new primary. Concurrency makes it hard and potentially a bigger problem. Maybe say "This is caused by operations that were in flight when the primary failed and were not yet processed by both shard copies"